Decred-Never-Stops.md

```diff
@@ -55,12 +55,14 @@ At its core, Decred was envisioned not just as another digital currency, but as
 
 Decred's journey isn't just about creating a new form of currency; it is about building an ecosystem where innovation thrives. Decred's hybrid model not only ensures security and decentralization but also fosters a culture of continuous development and adaptation. Let's check what makes Decred a unique project!
 
-A Legacy of Innovation
+## A Legacy of Innovation
 
 The innovation lies at the core of the Decred project, driving its pursuit of progress in the blockchain ecosystem. Formed by a team of exceptional developers, where many have worked for big companies and coded for big projects, such as Bitcoin, Decred stands out as a fertile ground for forward-thinking solutions and blockchain applications. Decred's unique hybrid consensus model is central to its ability to stay at the forefront of innovation which enables swift adoption of new technologies and methodologies.
 
 Decred's network operates on a unique hybrid consensus protocol, blending two key approaches: Proof of Work (PoW) and Proof of Stake (PoS). In this setup full nodes meticulously compile transaction lists for future blocks, ensuring adherence to specified requirements such as fees, fund availability, and prevention of double spending. Meanwhile, miners engage in solving cryptographic puzzles, aiming to construct the subsequent block. Upon puzzle solution, miners incorporate as many transactions as feasible from the network's full nodes into the block. This block is then introduced to the Decred blockchain as unmined, pending verification. Up to five randomly selected Proof-of-Stake (POS) tickets then scrutinize the block against consensus rules, and upon successful validation, it is officially added to the blockchain.
 
+Employing a Hybrid Proof-of-Work/Proof-of-Stake (PoW/PoS) consensus mechanism presents numerous advantages over alternative consensus models within the Decred ecosystem. Firstly, it effectively tackles scalability and energy-related issues commonly associated with pure Proof-of-Work systems. Secondly, it mitigates the risk of blockchain attacks by validators possessing significant coin holdings, a concern prevalent in solely Proof-of-Stake networks.
```

Comment:
"it effectively tackles scalability" I don't think this is true. In terms of scaling, what do you mean?